song-server/src/main/java/bio/overture/song/server/controller/analysisType/AnalysisTypePageableResolver.java

```diff
@@ -59,7 +58,7 @@
   public static final String DEFAULT_SORT_VARIABLE = VERSION;
   public static final Direction DEFAULT_SORT_ORDER = DESC;
 
-  private static final String ALLOWED_SORT_VARIABLES = COMMA.join(VERSION, NAME);
+  private static final String ALLOWED_SORT_VARIABLES = VERSION + "," + NAME;
```

Comment:
Create a no args constuctor `Joiners` class that contains  the following:
`public static final Joiner COMMA = Joiner.on(",");`
likewise for any other characters to join on